microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

```diff
@@ -12,12 +12,59 @@ ms.date: 02/23/2022
 This page lists the updates to Progressive Web Apps (PWAs) and Web Apps from the Microsoft Edge Web Apps team.  To try new features, read these announcements.  To stay up to date with the latest and greatest features, download the [Microsoft Edge preview channels](https://www.microsoftedgeinsider.com/download) (Beta, Dev, and Canary).
 
 
+<!-- ====================================================================== -->
+## What's New in Microsoft Edge 105
+
+Microsoft Edge 105 brings the following new features for Progressive Web Apps.
+
+
+### Window Controls Overlay
+
+The **Window Controls Overlay** API is now available by default with Microsoft Edge 105. This feature makes it possible for web developers to take control of the area that would normally be used by the title bar of an installed PWA window. The PWA code can use this area to render arbitrary web content such as a custom title, navigation buttons, or a menu bar.
```

Comment:
```suggestion
The **Window Controls Overlay** API is now available by default in Microsoft Edge 105. The Window Controls Overlay API enables your app to control the area that would normally be used by the title bar of an installed PWA window. The PWA code can use this area to render arbitrary web content such as a custom title area, navigation buttons, or a menu bar.
```
looking for alternative to broad words 'using' and 'with' that's more specific/narrow
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with

Although it is good sometimes to differentiate actors (user vs web dev'r), favor "you"; 2nd-person per Style, speaking directly to the defined audience except where contrasting roles of you the dev'r vs. your users of your app.
https://learn.microsoft.com/en-us/style-guide/grammar/person#in-general-use-second-person-pronouns-you-your

Replaced abstract/indirect "this feature" by a self-contained definition statement.